client/src/components/tools/item.jsx

```diff
@@ -178,12 +178,15 @@ export default ({ history, itemname, setLoading }) => {
           )}
         </div>
         <div className="eden_item-description">
-          {item.desc.split('\n').map((s, i) => (
-            <p key={`desc_ln_${i}`}>
-              {descriptionWithElements(s, i)}
-              <br />
-            </p>
-          ))}
+          {item.desc !== null &&
+            typeof item.desc !== 'undefined' &&
```

Comment:
Should be able to change these two lines to just one `item.desc &&`, since that will also ensure that it is non-null, non-undefined, and that it is a non-empty string.

Reply:
Oh wow, i didn't know that check did all that. Thanks for the tip. I'll make that update.